deploy/charts/cert-manager/templates/service.yaml

```diff
@@ -19,6 +19,7 @@ spec:
   ports:
     - protocol: TCP
       port: 9402
+      name: {{ .Values.prometheus.servicemonitor.portName }}
```

Comment:
question: Is there some reason why we might need this name to be configurable? Or is it just added here as a "nice-to-have"?

It seems on the face of it we could just hardcode this to `tcp-prometheus-servicemonitor` (which I just tested locally) and that'd work, to both make it very clear what the port is for and to avoid further complicating the configuration options we expose for the helm chart?

Or is there maybe a prometheus reason why someone would want to change this?

(I can see from the linked [Istio docs](https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/#explicit-protocol-selection) in the PR that it needs to start `tcp-`)

Reply:
It was just added as "nice-to-have", but doesn't seems necessary to have this name as a parameter. I've hardcoded it to avoid complicating the configuration.